dotcom-rendering/scripts/json-schema/schema.js

```diff
@@ -0,0 +1,33 @@
+const TJS = require('typescript-json-schema');
+const path = require('path');
+
+const root = path.resolve(__dirname, '..', '..');
+
+const program = TJS.getProgramFromFiles(
+	[
+		path.resolve(`${root}/src/lib/content.d.ts`),
+		path.resolve(`${root}/index.d.ts`),
+	],
+	{
+		skipLibCheck: true,
+	},
+);
+
+const settings = { rejectDateType: true, required: true };
+
+module.exports = {
+	generateArticleSchema: () => {
```

Comment:
Just a naming issue, we seem to be overloading `gen` / `generate`. We use `gen` / `generate` here to refer to creating a string from TS. We then use `gen` to refer to writing the files to disk.

Not a biggy, but would `gen-schema` and  `write-schema`, or `get-schema` and `gen-schema` be more clear>?